docroot/profiles/lagunita/summer_profile/modules/summer_helper/src/Plugin/paragraphs/Behavior/SummerCardBehaviors.php

```diff
@@ -38,6 +38,19 @@ public function buildBehaviorForm(ParagraphInterface $paragraph, array &$form, F
       ],
       '#default_value' => $paragraph->getBehaviorSetting('su_card_styles', 'sum_card_variant'),
     ];
+    $element['sum_card_bg_color_variant'] = [
+      '#type' => 'select',
+      '#title' => $this->t('Card Background Color'),
```

Comment:
```suggestion
      '#title' => $this->t('Pill Card Background Color'),
```
@mdyoung3 should we call this 'Pill Card Background Color' instead? So content authors know it's for the Pill Card only 😃  Unless we can conditional render this select field to only display when the user selects Pill from the Card Style.